hikari/events/channel_events.py

```diff
@@ -279,6 +279,14 @@ class GuildChannelUpdateEvent(GuildChannelEvent, ChannelUpdateEvent):
     shard: gateway_shard.GatewayShard = attr.ib(metadata={attr_extensions.SKIP_DEEP_COPY: True})
     # <<inherited docstring from ShardEvent>>.
 
+    old_channel: typing.Optional[channels.GuildChannel] = attr.ib(repr=True)
+    """Old guild channel object from cache or 'builtins.None'.
+
+    Returns
+    -------
+    hikari.channels.GuildChannel
+    """
```

Comment:
This doesnt need a return since its an attribute. Also, a better wording for the docstring is 
```suggestion
    """The old guild channel object.
    
    This will be `builtins.None` if it's missing from the cache.
    """
```
Again, this applies for all the other ones